lib/rules/value-no-vendor-prefix/index.js

```diff
@@ -59,11 +48,9 @@ const rule = (primary, secondaryOptions, context) => {
 		root.walkDecls((decl) => {
 			const { value } = decl;
 
-			if (
-				!isStandardSyntaxDeclaration(decl) ||
-				!isStandardSyntaxProperty(decl.prop) ||
-				!value.startsWith('-')
-			) {
+			if (!hasPrefix(value)) return;
```

Comment:
A more agnostic way would be to use
https://github.com/stylelint/stylelint/blob/3065425dc94be536dcf4369dcc1ed47735259e07/lib/utils/vendor.js#L22-L30
instead.